Model/Config/Source/MerchantAccounts.php

```diff
@@ -48,6 +48,6 @@ public function __construct(
     public function toOptionArray()
     {
         $merchantAccount = $this->_adyenHelper->getAdyenMerchantAccount('adyen_cc');
-        return $merchantAccount ? [$merchantAccount] : [];
+        return !empty($merchantAccount) ? array($merchantAccount => $merchantAccount) : [];
```

Comment:
We discussed this and found that this works, but now I also see that we did it differently for the ChangedCurrency https://github.com/Adyen/adyen-magento2/blob/main/Model/Config/Source/ChargedCurrency.php

Could you check if that approach also works here? It is important that this doesn't break in all M2 versions.

Reply:
Won't this return `['MyMerchantAccountName' => 'MyMerchantAccountName']`? I thought the idea was to return something like `['value' => 'MyMerchantAccountName', 'label' => 'Merchant Account']`

Reply:
It is modified. Check again ? @peterojo @acampos1916